spec/controllers/provision_controller_spec.rb

```diff
@@ -81,94 +78,4 @@
       end
     end
   end
-
-  describe 'DELETE' do
```

Comment:
do we want to have any tests in for when we receive a deprovision request?

Reply:
The only test case we have now that would remain is `returns with a 200 OK response`, and this is also verified in the cuke tests here: https://github.com/cyberark/conjur-service-broker/blob/master/features/deprovision.feature#L11.

I don't hold strongly one way or the other though, so I'm happy to leave that spec in.